scripts/pi-hole/js/queries.js

```diff
@@ -306,11 +324,7 @@ function formatInfo(data) {
   if (data.reply.type !== "UNKNOWN") {
     replyInfo = divStart + "Reply:&nbsp&nbsp;" + data.reply.type;
     if (data.reply.time >= 0 && data.reply.type !== "UNKNOWN") {
-      replyInfo +=
-        " (" +
-        (data.reply.time < 1
-          ? (1e3 * data.reply.time).toFixed(1) + " ms)"
-          : data.reply.time.toFixed(1) + " s)");
+      replyInfo += " (" + formatReplyTime(data, "display") + ")";
```

Comment:
I don't think this info is necessary. It only doubles the info we have already in the delay column